cms/templates/videos_index.html

```diff
@@ -6,7 +6,7 @@
   from django.utils.translation import ugettext as _
 %>
 <%block name="title">${_("Video Uploads")}</%block>
-<%block name="bodyclass">is-signedin course view-uploads view-video-uploads</%block>
+<%block name="bodyclass">is-signedin course view-video-uploads</%block>
```

Comment:
@clrux FYI, I figured it out! The class name here has to match the classname above in _header.scss.

This file incorrectly listed view-uploads in addition to view-video-uploads (causing them both to be highlighted once I fixed the bug in _header.scss that had a typo in the nav-course-courseware-videos classname).